File: benchmarks/php/recursive_fib.php
<?php
function fib(int $n): int {
    if ($n < 2) {
        return $n;
    }
    return fib($n - 1) + fib($n - 2);
}

$n = isset($argv[1]) ? (int) $argv[1] : 28;
echo fib($n), PHP_EOL;

File: benchmarks/python/recursive_fib.py
import sys


def fib(n: int) -> int:
    if n < 2:
        return n
    return fib(n - 1) + fib(n - 2)


n = int(sys.argv[1]) if len(sys.argv) > 1 else 28
print(fib(n))

